tree_hugger/core/parser/java/java_parser.py

```diff
@@ -48,9 +48,16 @@ def get_all_method_bodies(self) -> Dict[str, str]:
         captures = self._run_query_and_get_captures('all_method_bodies', self.root_node)
         
         method_bodies = {}
-        for i in range(0, len(captures), 2):
-            method_name = match_from_span(captures[i][0], self.splitted_code)
-            method_bodies[method_name] = match_from_span(captures[i+1][0], self.splitted_code)
+        i = 0
+        while i < len(captures):
```

Comment:
It probably can be re-written as 

```
for i, element in enumerate(captures):
```

Thik about it. It is probably more Pythonic

Reply:
Can't be done with a for loop : the incrementation depend on the type of the element